worlds/sc2/docs/setup_fr.md

```diff
@@ -0,0 +1,191 @@
+dossier `output/` de votre installation d'Archipelago.
+Il est recommandé de lancer `ArchipelagoGenerate.exe` via un terminal afin que vous puissiez voir les messages générés 
+par le logiciel, ce qui va inclure toutes erreurs qui ont eu lieu et le nom de fichier généré.
+Si vous n'appréciez pas le fait d'utiliser un terminal, vous pouvez aussi regarder le fichier *log* qui va être produit 
+dans le dossier `logs/`.
+
+#### À quoi sert l'option *Progression Balancing*?
+
+Pour un monde *Starcraft 2* seule, cette option ne fait rien. 
+Il s'agit d'une option d'Archipelago qui permet de balancer la progression d'un monde relativement aux autres mondes en 
+interchangeant les *items* de progression dans les *sphères*.
+Si le *Progression Balancing* d'un monde est plus grand que ceux des autres, les *items* de progression de ce monde ont 
+plus de chance d'être obtenus tôt et vice-versa si sa valeur est plus petite que celle des autres mondes.
+Cependant, *Starcraft 2* est beaucoup plus permissif en termes d'*items* qui sont nécessaires pour progresser.
+Pour cette raison, cet ajustement a souvent peu d'influence sur la capacité de progresser dans un monde de 
+*StarCraft 2*. 
+Notez que l'utilisation de cette option, i.e. au moins un monde avec une valeur différente de *Progression Balancing*, 
+augmente le temps de génération d'un *MultiWorld*, alors certains recommandent de ne pas toucher à cette option.
+
+#### Comment est-ce que je définis une liste d'*items*, e.g. pour l'option *excluded items*?
+
+Vous pouvez lire sur la syntaxe des conteneurs dans le format *yaml* à la page 
+[*YAML specification*](https://yaml.org/spec/1.2.2/#21-collections). 
+Pour les listes, chaque *item* doit être sur sa propre ligne et doit être précédé par un trait d'union.
+
+```yaml
+excluded_items:
+  - Battlecruiser
+  - Drop-Pods (Kerrigan Tier 7)
+```
+
+Une liste vide est représentée par une paire de crochets: `[]`. 
+Il s'agit de la valeur par défaut dans le modèle de base, ce qui devrait vous aider à apprendre à utiliser cette 
+syntaxe.
+
+#### Comment est-ce que je fais pour avoir des *items* dès le départ?
+
+L'option *starting inventory* est un *map* et non une liste. 
+Ainsi, elle permet de spécifier le nombre de chaque *item* avec lequel vous allez commencer.
+Sa syntaxe consiste à indiquer le nom de l'*item*, suivi par un deux-points, puis par un espace et enfin par le nombre 
+désiré de cet *item*.
+
+```yaml
+start_inventory:
+  Micro-Filtering: 1
+  Additional Starting Vespene: 5
+```
+
+Un *map* vide est représenté par une paire d'accolades: `{}`. 
+Il s'agit de la valeur par défaut dans le modèle de base, ce qui devrait vous aider à apprendre à utiliser cette 
+syntaxe.
+
+#### Comment est-ce que je fais pour connaître le nom des *items* dans *StarCraft 2 Archipelago*? 
+
+Vous trouverez la liste complète des *items* de *StarCraft 2 Archipelago* à la page 
+[*Icon Repository*](https://matthewmarinets.github.io/ap_sc2_icons/).
```

Comment:
pas de mention du data package?

Reply:
Hmm, well, I have do idea what that is and it is note in the English version. Do you have a ref? I will read on it and incorporate the info., if I can, in both language.

Reply:
Data package was mentioned. Since it also provides locations names, changed the title to include location. Since location is in the title, a mention that they are easy to find in the client was also made.

This was done in the French and English documention for uniformity.